stdlib/importlib/metadata/__init__.pyi

```diff
@@ -10,7 +10,7 @@ from pathlib import Path
 from typing import Any, ClassVar, Iterable, NamedTuple, Pattern, overload
 
 if sys.version_info >= (3, 10):
-    from importlib.metadata._meta import PackageMetadata as PackageMetadata
+    from importlib.metadata._meta import PackageMetadata
```

Comment:
Why make this change? It's present in the module's `__all__` (https://github.com/python/cpython/blob/be095f6c32188bba02079d086ac8639ea37cec3c/Lib/importlib/metadata/__init__.py#L33)

Reply:
You're right, I forgot that this is a stubs convention 😅 Will fix.

Reply:
(Imports in a stub are considered private to the stub unless you use the weird `import foo as foo` idiom — it's one of the ways stub files differ from `.py` files 🙂)

Reply:
Fixed.

Reply:
Technically type checkers use the `X as X` idiom when determining public symbols in .py code too: it's more a weird typed Python convention than a stubs convention

Reply:
Huh, TIL.